src/go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/accuknox/auto-policy-discovery/src
 go 1.20
 
 replace (
-	github.com/accuknox/auto-policy-discovery/src/protobuf => ./protobuf
```

Comment:
We should keep this line.
1. If someone pulls the DE repo and build it, it would be useful to get the latest probuf code directly from the relative path.
2. If someone import DE `src` pkg, then he/she will use the commitID mentioned in the `require` block.